### PR TITLE
Use the new build scripts for building/publishing images

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/platform/issues/5626

This is almost entirely copy/pasta from the API repo; the storage-service builds are pretty vanilla compared to the catalogue pipeline.

### CODE REVIEW

File: builds/build_sbt_image.sh [hunk 1]
```diff
@@ -1,24 +1,41 @@
 #!/usr/bin/env bash
+<<EOF
+Build the Docker image for an sbt project.
+
```
Comment (alexwlchan) [inline builds/build_sbt_image.sh:4]: copied from catalogue-api

File: builds/get_sbt_project_directory.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+<<EOF
+Finds the path to an sbt project directory.
```
Comment (alexwlchan) [inline builds/get_sbt_project_directory.sh:3]: copied from catalogue-api

File: builds/publish_sbt_image_to_ecr.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+<<EOF
```
Comment (alexwlchan) [inline builds/publish_sbt_image_to_ecr.sh:2]: copied from catalogue-api

File: builds/run_sbt_tests.sh [hunk 4]
```diff
@@ -1,21 +1,40 @@
 #!/usr/bin/env bash
+<<EOF
+Run tests for an sbt project.
```
Comment (alexwlchan) [inline builds/run_sbt_tests.sh:3]: copied from catalogue-api

File: builds/update_ecr_image_tag.sh [hunk 5]
```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+<<EOF
+Retag an image in ECR.
```
Comment (alexwlchan) [inline builds/update_ecr_image_tag.sh:3]: copied from catalogue-api

**Review by paul-butcher — APPROVED** [review]

**Pull request merged**